### ISSUE TITLE
Not all meetings showing on phone

### ISSUE THREAD

**Issue by ManFromNapa** [code4recovery/tsml-ui#284]
When I filter by a specific day, users are not seeing all the meetings are showing.  This only seems to happen on phones.  On my laptop, and iPad I see all the meetings, but on my iPhone I’m only seeing the first 10 meetings.  I also do not see an option for a “next page”.

In fact as I am writing this, I realize on my iPhone only shows the next 10 meetings with no option to see the next page.  Is there an option I’m missing?

**Comment by joshreisner** [comment]
it should load 10 more meetings when you scroll down, do you see any errors in the javascript console?

im away and dont have access to a desktop web browser so i cant investigate until next week

presumably this is not a widespread problem, you dont see it happening on aasfmarin.org/meetings, right?

**Comment by joshreisner** [comment]
looks like you’ve got a few floating javascript add-ons like language, privacy, and accessibility tools. were any of those added recently?

**Comment by ManFromNapa** [comment]
None of those were added recently.  I’ll check the JavaScript console for errors.  I’ll also try disabling plugins to see if one of those is causing this.  I’ll post an update with what I find.

**Comment by enderv** [comment]
Hooked up my phone to get logs and not seeing any errors in the console when I try to browse through the list, not sure what else to look for though

**Comment by joshreisner** [comment]
was the theme updated recently? it works on desktop? how about desktop with mobile viewport size?

**Comment by ManFromNapa** [comment]
I finally have some time to troubleshoot.  On my phone I see 10 meetings.  Desktop and iPad shows me 20.

I'm going to copy to a staging site and start disabling plugins to see if one of them is causing this.  Will post once I have some new data.

**Comment by ManFromNapa** [comment]
Disabled all plugins and cleared my browser cache.  Plugin behavior didn't change.

Trying to think about what could be creating this limit.  Open to any and all suggestions.

**Comment by joshreisner** [comment]
try changing the theme. sounds like something is preventing it from knowing it has scrolled down. some odd css could be at fault?

**Comment by ManFromNapa** [comment]
CSS... my favorite.  LOL. 

I'm looking into CSS now.  I'll try the theme a little later and update everyone.

**Comment by joshreisner** [comment]
presumably this was working before right?

**Comment by ManFromNapa** [comment]
I'm not sure to be honest.  I feel like this might have always been an issue, I just wasn't aware of it.  After looking at other goup's meeting page I don't remember ours ever listing as many meetings.

**Comment by joshreisner** [comment]
huh ok! we’ll definitely figure it out, but it might not be until i’m back in front of a computer tuesday night

**Comment by ManFromNapa** [comment]
Understood and appreciated.  I'll keep trying to debug, and try a new theme when I have the time.

**Comment by enderv** [comment]
tried a few more things: in mobile switch to desktop view works fine. from desktop chome trying to use the devtools to mimic phone everything works fine.

**Comment by ManFromNapa** [comment]
It's not limiting you to 20 meetings?  No matter what I try I can't get more than 20 meetings to show in desktop mode.

**Comment by enderv** [comment]
Didn't count but def more then 20 is showing am able to scroll a bit.

**Comment by ManFromNapa** [comment]
Well @#$^#... LOL. I've tried MacOS & Windows desktops.  iPhone and iPad (Chrome & Safari on both).  I'm seeing the limits.  Wish I could see what you see, and figure out what the difference is.

**Comment by ManFromNapa** [comment]
Set up my staging site.  Disabled ALL plugins and still had the limited # of meetings show.  Changed my theme and was able to see all my meetings.  Re-activated all my plugins, still saw all the meetings.  So it's my theme.  

Any recommendations on where to begin troubleshooting this?

**Comment by tim-rohrer** [comment]
> Set up my staging site. Disabled ALL plugins and still had the limited # of meetings show. Changed my theme and was able to see all my meetings. Re-activated all my plugins, still saw all the meetings. So it's my theme.
> 
> Any recommendations on where to begin troubleshooting this?

Look for errors in the console is where I would start. If present, open an issue with the theme developer. 

What theme are you trying to use?

**Comment by ManFromNapa** [comment]
I’m using Inspiro.  I’ll open a ticket with them and share a link here for informational purposes, and in case anyone else experiences the same issue.

**Comment by ManFromNapa** [comment]
Nothing comes up on the console, but I did discover something.  As I "zoom out" and the font gets smaller, more results show.  That is, as the resolution changes so does the number of results.  I'm investigating the theme settings about this.

**Comment by ManFromNapa** [comment]
[https://wordpress.org/support/topic/page-resolution-effects-js-results/]

**Comment by tim-rohrer** [comment]
I'm convinced this isn't a TSML UI issue, and I'm going to close this issue.

If needed, we can reopen later.

**Comment by ManFromNapa** [comment]
> I'm convinced this isn't a TSML UI issue, and I'm going to close this issue.
> 
> If needed, we can reopen later.

I agree.  I’ll keep updating this as I find a solution to help others who have this issue.

**Comment by joshreisner** [comment]
@ManFromNapa i'm finally back in front of a computer. try adding this custom CSS to your site:

```
html, body { height: auto !important; }
```

**Comment by ManFromNapa** [comment]
> html, body { height: auto !important; }

That appears to have resolved the issue.  Let me do some more testing with different browers, OS, and devices.  Thank you all!!!
